### PR TITLE
EWLJ-764: Updated tab functionality to handle multiple sets of tabs.

### PR DESCRIPTION
**Pull request by jorge-melo**
**Jira Ticket**

- [EWLJ-764: (Mobile) Video Tabs component](https://ama-it.atlassian.net/browse/EWLJ-764)

## Description

Given: Individual has accessed JOE website through mobile device. When the user tries to view a video in the Video Tabs component Then the system should correct the aspect ratio on page load without any interaction from the user.

Current Behavior

In mobile - When first scrolling to the Video Tabs component, the preview image displayed for the first video in the tabs is not sized properly/is cut off (see firest screenshot below). Once you click on another video tab and then click back to the video 1 title tab, it resizes correctly (see second screenshot below).

## To Test

- [x] Create an "Art of medicine" content type.
- [x] Make sure to include some videos on "body content" tab under "body component" area.

Screenshot_13-11-2024_183957_journalofethics lndo site

**Go to the page you just created and visualize it as Mobile device.**
Screenshot 2024-11-13 at 18-43-43 Video tabs heading text test Journal of Ethics American Medical Association

**Both video sets should have the same aspect ratio.**

### CODE REVIEW

**Comment by isa-ama** [comment]
@jorge-melo  Issue still there.  Use browserstack to fix the bug.

Iphonebug1

**Comment by jorge-melo** [comment]
Screenshot_15-11-2024_105917_live browserstack com

**Review by isa-ama — APPROVED** [review]

**Comment by isa-ama** [comment]
looks good.

**Comment by isa-ama** [comment]
desk1

Make sure the code only applies to **Mobile**

File: styleguide/source/assets/js/vc-video-tabs.js [hunk 1]
```diff
@@ -38,54 +45,64 @@
...
 
-        // Initially activate the first tab and reveal the first tab panel
+        // EWLJ-764: Initially activate the first tab and reveal the first tab panel
         tabs[0].removeAttribute('tabindex');
         tabs[0].setAttribute('aria-selected', 'true');
         panels[0].hidden = false;
-      }
+        
+        // EWLJ-764: Force a resize after the first load to ensure the correct aspect ratio
+        window.addEventListener('load', () => {
```
Comment (isa-ama) [inline styleguide/source/assets/js/vc-video-tabs.js]: This should apply only for mobile

**Review by isa-ama — CHANGES_REQUESTED** [review]
The code should apply only for mobile.

**Comment by isa-ama** [comment]
roateIphone

Issue still there when the phone is rotated.

**Review by isa-ama — APPROVED** [review]

**Comment by isa-ama** [comment]
Looks good to me.

**Review by rgb-ama — APPROVED** [review]
approved

**Pull request merged**